core/liblwm2m.h

```diff
@@ -520,7 +526,7 @@ int lwm2m_configure(lwm2m_context_t * contextP, char * endpointName, char * msis
 int lwm2m_start(lwm2m_context_t * contextP);
 
 // check if the server registrations are outdated and needs to be renewed
-int lwm2m_update_registrations(lwm2m_context_t * contextP, uint32_t currentTime, struct timeval * timeoutP);
+int lwm2m_update_registrations(lwm2m_context_t * contextP, time_t currentTime, time_t * timeoutP);
```

Comment:
Actually, this function should update the timeout if a registration is needed. I will change this